<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/* Основные настройки фона и текста */
QWidget {
    background-color: #272727; /* фон для всего приложения */
	font-family: &quot;GOST Common&quot;;
	text-align: center;
    color: #eeeeee;            /* цвет текста */
    font-size: 14px;           /* Стандартный размер текста */
}

/* Настройки для кнопок */
QPushButton {
    background-color: #5b5b85;     /* Основной цвет */
    color: white;                  /*текст */
    border-radius: 10px;           /* Скругленные углы */
    padding: 8px 16px;             /* Отступы */
    border: 2px;     /* Бордер с ярким акцентом */
	box-shadow: 2px 2px 5px rgba(0, 0, 0, 0.5);
}

QPushButton:hover {
    background-color: #aaaaff;     /* Яркий розовый цвет при наведении */
}

QPushButton:pressed {
    background-color: #aaaaff;     /* Красный цвет при нажатии */
    border: 2px solid;     /* Оранжевая граница при нажатии */
	box-shadow: inset 2px 2px 5px rgba(0, 0, 0, 0.5);
}

/* Настройки для слайдеров */
QSlider::groove:horizontal {
    background: #eeeeee;           /* трек */
    height: 6px;
    border-radius: 3px;
}

QSlider::handle:horizontal {
    background: #aaaaff;           /* ручка */
    border: 2px solid #5b5b85;     /* граница */
    width: 20px;
    height: 20px;
    margin: -8px 0;
    border-radius: 10px;
}

QSlider::handle:horizontal:hover {
    background: #5b5b85;           /* цвет при наведении */
}

/* Настройки для полей ввода текста */
QLineEdit {
    background-color: #2C2C2C;     /* Темный фон */
    color: #eeeeee;                  /* Белый текст */
    border: 1px solid #5b5b85;     /* Фиолетовая рамка */
    padding: 6px;
    border-radius: 6px;
}

QLineEdit:focus {
    border: 1px solid #D900FF;     /* Яркий розовый при фокусе */
}

/* Настройки для выпадающих списков */
QComboBox {
    background-color: #2C2C2C;     /* Темный фон */
    color: white;                  /* Белый текст */
    border: 1px solid #FF1F00;     /* Красная рамка */
    padding: 6px;
    border-radius: 6px;
}

QComboBox QAbstractItemView {
    background-color: #2C2C2C;     /* Темный фон выпадающего списка */
    color: white;                  /* Белый текст */
    selection-background-color: #A63BFF; /* Фиолетовый при выборе */
}

/* Настройки для чекбоксов */
QCheckBox {
    color: white;
}

QCheckBox::indicator {
    width: 18px;
    height: 18px;
    border-radius: 3px;
    background-color: #2C2C2C;
    border: 2px solid #FF1F00;     /* Красная рамка */
}

QCheckBox::indicator:checked {
    background-color: #A63BFF;     /* Фиолетовый цвет при выборе */
    border: 2px solid #D900FF;     /* Яркая розовая рамка при выборе */
}

/* Настройки для радиокнопок */
QRadioButton {
    color: white;
}

QRadioButton::indicator {
    width: 16px;
    height: 16px;
    border-radius: 8px;
    background-color: #2C2C2C;
    border: 2px solid #FF1F00;     /* Красная рамка */
}

QRadioButton::indicator:checked {
    background-color: #A63BFF;     /* Фиолетовый цвет при выборе */
    border: 2px solid #D900FF;     /* Розовая рамка при выборе */
}
QTabWidget {
    background-color: #2D2D30; /* Цвет фона всего виджета */
    color: #eeeeee; /* Цвет текста */
	border: none;
}

QTabBar {
    background: #313131; /* Цвет фона для панели вкладок */
    border: none; /* Цвет границы панели вкладок */
}

QTabBar::tab {
    background: #5b5b85; /* Цвет фона вкладок */
    color: #eeeeee; /* Цвет текста вкладки */
    padding: 3px; /* Отступы внутри вкладки */
    margin: 0 2px; /* Отступы между вкладками */
    border: 1px solid #313131; /* Цвет границы вкладок */
    border-radius: 5px; /* Скругление углов вкладок */
}

QTabBar::tab:selected {
    background: #aaaaff; /* Цвет фона выбранной вкладки */
    color: #eeeeee; /* Цвет текста выбранной вкладки */
    border-bottom: none; /* Убираем границу внизу для выбранной вкладки */
    border-top-left-radius: 5px; /* Скругление верхнего левого угла для выбранной вкладки */
    border-top-right-radius: 5px; /* Скругление верхнего правого угла для выбранной вкладки */
    border-bottom-left-radius: 0px; /* Убираем скругление нижнего левого угла */
    border-bottom-right-radius: 0px; /* Убираем скругление нижнего правого угла */
}

QTabBar::tab:hover {
    background: #aaaaff; /* Цвет фона при наведении на вкладку */
}
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>801</width>
      <height>571</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="mainMenu">
     <attribute name="title">
      <string>menu</string>
     </attribute>
     <widget class="QWidget" name="gridLayoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>112</width>
        <height>161</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="ButtonsGridLayout">
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="spacing">
        <number>0</number>
       </property>
       <item row="3" column="0">
        <widget class="QPushButton" name="getDevicesButton">
         <property name="text">
          <string>Show devices</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QPushButton" name="getDataButton">
         <property name="text">
          <string>Show charts</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="connectButton">
         <property name="text">
          <string>Connect</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QPushButton" name="settingsButton">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>Settings</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="stateLabel">
      <property name="geometry">
       <rect>
        <x>638</x>
        <y>0</y>
        <width>131</width>
        <height>33</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Disconnected&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="rebootButton">
      <property name="geometry">
       <rect>
        <x>690</x>
        <y>40</y>
        <width>75</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    background-color: #e0301e;     /* Основной цвет */
    color: #eeeeee;                  /*текст */
    border-radius: 10px;           /* Скругленные углы */
    padding: 4px 8px;             /* Отступы */
    border: 2px;     /* Бордер с ярким акцентом */
}

QPushButton:hover {
    background-color: #eb8c00;     /* Яркий розовый цвет при наведении */
}

QPushButton:pressed {
    background-color: #a32020;     /* Красный цвет при нажатии */
    border: 2px solid;     /* Оранжевая граница при нажатии */
}</string>
      </property>
      <property name="text">
       <string>Reboot</string>
      </property>
     </widget>
     <widget class="QWidget" name="gridLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>30</y>
        <width>421</width>
        <height>101</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="systemControl">
       <item row="2" column="1">
        <widget class="QPushButton" name="dbRestartBtn">
         <property name="styleSheet">
          <string notr="true">QPushButton {
    background-color: #ff7400;     /* Основной цвет */
    color: #eeeeee;                  /*текст */
    border-radius: 10px;           /* Скругленные углы */
    padding: 8px 16px;             /* Отступы */
    border: 2px;     /* Бордер с ярким акцентом */
}

QPushButton:hover {
    background-color: #ff9a00;     /* Яркий розовый цвет при наведении */
}

QPushButton:pressed {
    background-color: #ff4d00;     /* Красный цвет при нажатии */
    border: 2px solid;     /* Оранжевая граница при нажатии */
}</string>
         </property>
         <property name="text">
          <string>restart</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QPushButton" name="brokerRestartBtn">
         <property name="styleSheet">
          <string notr="true">QPushButton {
    background-color: #ff7400;     /* Основной цвет */
    color: #eeeeee;                  /*текст */
    border-radius: 10px;           /* Скругленные углы */
    padding: 8px 16px;             /* Отступы */
    border: 2px;     /* Бордер с ярким акцентом */
}

QPushButton:hover {
    background-color: #ff9a00;     /* Яркий розовый цвет при наведении */
}

QPushButton:pressed {
    background-color: #ff4d00;     /* Красный цвет при нажатии */
    border: 2px solid;     /* Оранжевая граница при нажатии */
}</string>
         </property>
         <property name="text">
          <string>restart</string>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QLabel" name="noneLabel">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;TextLabel&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="mqttBrokerStateLabel">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;active&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QPushButton" name="rebootAllDevicesBtn">
         <property name="styleSheet">
          <string notr="true">QPushButton {
    background-color: #ff7400;     /* Основной цвет */
    color: #eeeeee;                  /*текст */
    border-radius: 10px;           /* Скругленные углы */
    padding: 8px 16px;             /* Отступы */
    border: 2px;     /* Бордер с ярким акцентом */
}

QPushButton:hover {
    background-color: #ff9a00;     /* Яркий розовый цвет при наведении */
}

QPushButton:pressed {
    background-color: #ff4d00;     /* Красный цвет при нажатии */
    border: 2px solid;     /* Оранжевая граница при нажатии */
}</string>
         </property>
         <property name="text">
          <string>restart all</string>
         </property>
        </widget>
       </item>
       <item row="2" column="3">
        <widget class="QPushButton" name="pushButton_4">
         <property name="styleSheet">
          <string notr="true">QPushButton {
    background-color: #ff7400;     /* Основной цвет */
    color: #eeeeee;                  /*текст */
    border-radius: 10px;           /* Скругленные углы */
    padding: 8px 16px;             /* Отступы */
    border: 2px;     /* Бордер с ярким акцентом */
}

QPushButton:hover {
    background-color: #ff9a00;     /* Яркий розовый цвет при наведении */
}

QPushButton:pressed {
    background-color: #ff4d00;     /* Красный цвет при нажатии */
    border: 2px solid;     /* Оранжевая граница при нажатии */
}</string>
         </property>
         <property name="text">
          <string>restart</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QLabel" name="devicesCountLabel">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&amp;quot;count&amp;quot;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLabel" name="dbStateLabel">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;active&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;mqtt broker&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;database&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;devices&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;TextLabel&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="chartMenu">
     <attribute name="title">
      <string>charts</string>
     </attribute>
     <widget class="QWidget" name="verticalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>781</width>
        <height>521</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="chartLayout"/>
     </widget>
    </widget>
    <widget class="QWidget" name="controlMenu">
     <attribute name="title">
      <string>devices</string>
     </attribute>
     <widget class="QWidget" name="gridLayoutWidget_3">
      <property name="geometry">
       <rect>
        <x>-1</x>
        <y>-1</y>
        <width>791</width>
        <height>521</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="devicesLayout"/>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
